app/src/main/java/org/kiwix/kiwixmobile/settings/KiwixPrefsFragment.kt

```diff
@@ -23,7 +23,7 @@ import android.os.Environment
 import androidx.preference.Preference
 import org.kiwix.kiwixmobile.core.settings.CorePrefsFragment
 import org.kiwix.kiwixmobile.core.utils.SharedPreferenceUtil
-import org.kiwix.kiwixmobile.core.utils.SharedPreferenceUtil.PREF_STORAGE
+import org.kiwix.kiwixmobile.core.utils.SharedPreferenceUtil.Companion.PREF_STORAGE
```

Comment:
this change will be necessary if we declare the `const`s as top level

Reply:
* No, i don't think so. As to access constants declared inside _companionObject_, we'll need to explicitly use the `Companion` in import.
* If we had declared those static constants at top level, then this import(added one) won't be needed.
* Please correct, if i missed something or stated something wrong.

Reply:
I had a typo, *this change will not be necessary

declare them as top level `const`s, if a file doesn't have to change then don't change it, helps keep reviews focused

Reply:
Well, i couldn't avoid the editing in this file. As when I moved these constants to top level, then:
* It didn't let me make it static(i.e., it asked for using _object_ instead of a _CompanionObject_)
* Doing so, also lead to the decreased scope of the consts which I didn't prefer in order to maintain sync with the previous file.

Reply:
I was getting confused between java and kotlin imports, apologies. This way is best for interop